### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by akhil-8607**
updated the static UI according to https://issues.apache.org/jira/browse/AIRAVATA-3314 . 

More Improvements to be done in css

### CODE REVIEW

File: airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
     <div id="storage">
-        <div class="container">
+        <div class="container" style="margin-left:15rem;">
         <div class="row">
-            <div class="col-8">
+            <div class="col-4">
                 <h1 class="h4 mb-4">Storage Units</h1>
             </div>
-            <!-- Add buttons here -->
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (aarushiibisht) [inline airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue:8]: correct the indentation

File: airavata_mft/templates/base.html [hunk 2]
```diff
@@ -1,14 +1,85 @@
...
+    <!-- Custom styles -->
+    <link href="{% static 'base.css' %}" type="text/css" rel="stylesheet">
+  </head>
+
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light ">
+      <a class="navbar-brand" href="#">
+          AIRAVATA MFT
+      </a>
+   <div>
```
Comment (aarushiibisht) [inline airavata_mft/templates/base.html:29]: Can you please correct the indentation

File: airavata_mft/templates/base.html [hunk 3]
```diff
@@ -1,14 +1,85 @@
...
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Airavata MFT</title>
+    <!--  Bootstrap CSS CDN -->
```
Comment (aarushiibisht) [inline airavata_mft/templates/base.html:18]: Do not use bootstrap CDN. Instead include it in package.json file

File: airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+
+<div style="position: absolute; right: 70px;">
```
Comment (aarushiibisht) [inline airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue:10]: correct the indentation

File: airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+<button type="button" class="btn btn-outline-secondary "> Sort By  </button>
+
+<button class="btn" onclick="listView()"><i class="fa fa-bars"></i> </button>
```
Comment (aarushiibisht) [inline airavata_mft/apps/workspace/static/airavata_mft_workspace/js/containers/Storage.vue:13]: Cannot find listView() and gridView() functions

**Comment by akhil-8607** [comment]
closing this PR since this update is developed and merged

**Pull request closed without merging**